src/sentry/workflow_engine/migration_helpers/rule.py

```diff
@@ -175,21 +166,17 @@ def update_migrated_issue_alert(rule: Rule, data: UpdatedIssueAlertData):
     delete_workflow_actions(if_dcg=if_dcg)
     create_workflow_actions(if_dcg=if_dcg, actions=data["actions"])  # action(s) must exist
 
-    updated_data: dict[str, Any] = {
-        "name": data["name"],
-        "environment_id": data["environment"],
-        "enabled": True,
-        "owner_team_id": None,
-        "owner_user_id": None,
-        "config": {"frequency": data["frequency"] or Workflow.DEFAULT_FREQUENCY},
-    }
-    owner = data["owner"]
-    if owner:
-        if owner and owner.is_user:
-            updated_data["owner_user_id"] = owner.id
-        if owner and owner.is_team:
-            updated_data["owner_team_id"] = owner.id
-    workflow.update(**updated_data)
+    workflow.environment_id = rule.environment_id
+    if frequency := data["frequency"]:
+        workflow.config["frequency"] = frequency
+
+    workflow.owner_user_id = rule.owner_user_id
+    workflow.owner_team_id = rule.owner_team_id
+
+    workflow.name = rule.label
+
+    workflow.enabled = True
+    workflow.save()
```

Comment:
do we have any post_save signals we're relying on? if not, `.update` is generally more efficient, or adding a `update_fields` param to `.save`. 

Also nit: I find the previous approach of passing a dict of fields to update slightly easier to read.

Reply:
yes, we have this `post_save` signal. agree that the dict would have been better but i don't think it's possible with `save()`

https://github.com/getsentry/sentry/blob/6c8211bf06f2c4054369cafbb45dfd3feb001fd5/src/sentry/workflow_engine/models/workflow.py#L97-L99